models/nxos/lldp/deleted_example_01.txt

```diff
@@ -0,0 +1,17 @@
+# Using deleted
+# Before state:
+# -------------
+#
+#feature lldp
+#lldp holdtime 5
+#lldp reinit 3
+
+- name: Delete LLDP configuration (this will by default remove all lldp configuration)
```

Comment:
Does this stop the lldp service?

Reply:
Yes, if it defaulted to the previous config.

Reply:
@adharshsrivatsr the lldp feature should be disabled when deleted.